src/crypto/utils.ts

```diff
@@ -0,0 +1,96 @@
+import { bech32 as Bech32 } from "bech32";
+// @ts-ignore types are not available atm
+import createBlakeHash from "blake-hash";
+import { blake2bHex } from "blakejs";
+import JsSHA from "jssha";
+import createKeccakHash from "keccak";
+
+export const numberToHex = (number: number): string => {
+  let hex = Math.round(number).toString(16);
+
+  if (hex.length === 1) {
+    hex = "0" + hex;
+  }
+
+  return hex;
+};
+
+// TODO refactor
```

Comment:
is it refactored?

Reply:
I didn't want to make refactor in this PR, because finding a potential error will be much harder